internal/grafanadi/service/podphase.go

```diff
@@ -124,7 +126,7 @@ func ConvertPodPhase2Frame(podPhases []*storagemodel.LifePhase) model.DataFrame
 			op = phaseReason
 			t = "Event"
 		}
-		tags := fmt.Sprintf("\"%s\" UA=\"%s\" Level=\"%s\"", op, userAgent, state)
+		tags := fmt.Sprintf("HyperEvent=\"%s\" UA=\"%s\" Level=\"%s\"", op, userAgent, state)
```

Comment:
I don't think label it as HyperEvent is a good idea since HyperEvent is a internal concept, user may not know it.

Reply:
ok, i have deleted this label